### ISSUE TITLE
Unicode in queries

### ISSUE THREAD

**Issue by djordjeglbvc** [msiemens/tinydb#28]
When testing db field with unicode string, `UnicodeEncodeError` exception is raised.
Line which causes the exception:

``` python
db.get(where('name') == u'žir')
```

Inserting unicode data went without problems:

``` python
db.insert({'name': 'žir'})
```

I have made quick hack which fixes problem for my little hobby project, but I will examine this problem more when I find time.

In `queries.py`, I've changed `Query._update_repr` function body to:

``` python
self._repr = u'\'{0}\' {1} {2}'.format(self._key, operator, value)
```

and `Query.__hash__` to:

``` python
return hash(repr(unicode(self)))
```

Basically adding string preffix "u" in `_update_repr`, and "unicode" call in `__hash__`...

Using tinydb from git on python 2.7.6, ubuntu 14.04
